README.md

```diff
@@ -347,4 +392,18 @@ class MyCustomApplicationLoader extends ApplicationLoader {
     (new GuiceApplicationLoader).load(newContext)
   }
 }
-```
\ No newline at end of file
+```
+
+#### Play 2.3
+
+```scala
+import play.api._
+import com.typesafe.conductr.bundlelib.play.Env
+
+object Global extends GlobalSettings {
+  val totalConfiguration = super.configuration ++ Configuration(Env.asConfig)
```

Comment:
Probably better to override `onLoadConfig`:

```scala
override def onLoadConfig(config: Configuration, path: File, classloader: ClassLoader, mode: Mode.Mode) =
  config ++ Configuration(Env.asConfig)
```

`configuration` is only used by the default implementation of `onLoadConfig`. (I know, it's confusing...)